x-pack/plugins/osquery/public/routes/saved_queries/list/index.tsx

```diff
@@ -125,12 +125,12 @@ const SavedQueriesPageComponent = () => {
   );
 
   const renderPlayAction = useCallback(
-    (item: SavedQuerySO) => (
-      <PlayButton
-        savedQuery={item}
-        disabled={!(permissions.runSavedQueries || permissions.writeLiveQueries)}
-      />
-    ),
+    (item: SavedQuerySO) =>
+      permissions.runSavedQueries || permissions.writeLiveQueries ? (
```

Comment:
you should use `available` prop instead 

![image](https://user-images.githubusercontent.com/5188868/159889464-b10ea291-0fa9-42a9-9c05-e22db039d0c9.png)